andela/vali-ah-frontend
# #167377484 End to end testing using cypress.io

#### What does this PR do?
This pull request configures end to end testing using cypress.io
#### Description of Task to be completed
 - install cypress
 - add cypress test to circleci.yml
#### How should this be manually tested?
- git clone the branch
- cd into the root folder and run ```npm run cy:open```
- cypress should open and run the test
#### Any Background Context You want to provide?
N/A
#### What are the relevant pivotal tracker stories?

[#167377484](https://pivotaltracker.com/story/show/167377484)

#### Screenshots (if appropriate)
<img width="1189" alt="Screenshot 2019-08-26 at 5 44 08 PM" src="https://user-images.githubusercontent.com/34121552/63709428-c9ba6e80-c82e-11e9-8865-9dabf3247326.png">

#### Questions:
N/A
